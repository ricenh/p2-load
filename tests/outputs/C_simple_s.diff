--- outputs/C_simple_s.txt	2023-10-05 22:26:07.562570795 -0400
+++ expected/C_simple_s.txt	2023-09-19 13:29:21.000000000 -0400
@@ -1,4 +1,3 @@
-Usage: ../y86 <option(s)> mini-elf-file
- Options are:
-  -h      Display usage
-  -H      Show the Mini-ELF header
+ Segment   Offset    Size      VirtAddr  Type      Flags
+  00       0x0038    0x0015    0x0100    CODE      R X
+  01       0x004d    0x000b    0x0200    DATA      RW 
