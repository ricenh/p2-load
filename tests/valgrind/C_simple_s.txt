==1045370== Memcheck, a memory error detector
==1045370== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1045370== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==1045370== Command: ../y86 -s inputs/simple.o
==1045370== 
../y86: invalid option -- 's'
Usage: ../y86 <option(s)> mini-elf-file
 Options are:
  -h      Display usage
  -H      Show the Mini-ELF header
==1045370== 
==1045370== HEAP SUMMARY:
==1045370==     in use at exit: 0 bytes in 0 blocks
==1045370==   total heap usage: 1 allocs, 1 frees, 8,192 bytes allocated
==1045370== 
==1045370== All heap blocks were freed -- no leaks are possible
==1045370== 
==1045370== For lists of detected and suppressed errors, rerun with: -s
==1045370== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
