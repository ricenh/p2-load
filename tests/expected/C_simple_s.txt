 Segment   Offset    Size      VirtAddr  Type      Flags
  00       0x0038    0x0015    0x0100    CODE      R X
  01       0x004d    0x000b    0x0200    DATA      RW 
